pom.xml

```diff
@@ -228,14 +228,14 @@
         <archunit.version>0.14.1</archunit.version>
         <logback.version>1.2.3</logback.version>
         <jackson.version>2.10.1</jackson.version>
-        <jackson.databind.version>2.10.3</jackson.databind.version>
+        <jackson.databind.version>2.9.10.8</jackson.databind.version>
```

Comment:
This is a downgrade?

Reply:
@saudet  ah good catch, jackson-core and databind are both at 2.12.2. Just pushed the change there.